smf/cockroachdb/manifest.xml

```diff
@@ -4,7 +4,8 @@
 <service_bundle type='manifest' name='cockroachdb'>
 
   <service name='system/illumos/cockroachdb' type='service' version='1'>
-  <create_default_instance enabled='false' />
+  <create_default_instance enabled='true' />
+  <single_instance />
```

Comment:
The `<single_instance/>` node is obsolete; see https://www.illumos.org/issues/15477

Reply:
Removed